web/public_html/index.html

```diff
@@ -212,12 +212,9 @@
         </div>
       </main>
 
-      <iframe
-        name="torrent-upload-frame"
-        id="torrent-upload-frame"
-        src="about:blank"
-        title="Add Torrent"
-      ></iframe>
+      <div class="upload-div">
+        <h2>Drop torrent files here</h2>
```

Comment:
How about a more explicative message?
```suggestion
        <h2>Drag your torrent files here to add them to Transmission.</h2>
```

Reply:
Fine with me